objects/task5.js

```diff
@@ -0,0 +1,17 @@
+function multiplyNumeric(obj) {
+    for (key in obj) {
+        if (typeof obj[key] == 'number') {
+            console.log(key + ' is now ' + (obj[key] *= 2));
```

Comment:
not a good style, `obj[key] *= 2` move to separate line and after that output it if you want. Also you can use `console.info` instead of `console.log`

Reply:
separete this line